sdk/storage/Directory.Build.props

```diff
@@ -55,4 +55,7 @@
 
   <!-- Pull in Azure.Core helpers for TEST projects -->
   <Import Condition="'$(IsTestProject)' == 'true' and '$(IsClientLibrary)' == 'true'" Project="$(MSBuildThisFileDirectory)..\core\Azure.Core\tests\TestFramework.props" />
+  
+  <!--Add shared storage properties-->
+  <Import Project="$(MSBuildThisFileDirectory)\Azure.Storage.Common\src\Azure.Storage.Common.props" />
```

Comment:
Why not just define the values here?  Or if we really want a separate props file, let's pull it up to the top level.  Common.props looks like it should just apply to Common and I'd be confused why adding a value there affected all the other projects.

Reply:
Sure